src/lib/runDev.js

```diff
@@ -227,9 +226,15 @@ async function runDev (args = [], config, options = {}, log = () => {}) {
     log('press CTRL+C to terminate dev environment')
 
     if (config.app.hasBackend && fetchLogs) {
+      const pollArgs = {
+        config: devConfig,
+        logOptions: {
+          startTime: new Date().valueOf()
```

Comment:
The cool kids use `Date.now()` these days

Reply:
heh :) done